app/kumactl/cmd/install/install_control_plane.go

```diff
@@ -65,6 +67,9 @@ func newInstallControlPlaneCmd(pctx *kumactl_cmd.RootContext) *cobra.Command {
 			if err := core.ValidateCpMode(args.KumaCpMode); err != nil {
 				return err
 			}
+			if args.KumaCpMode == core.Local && args.ClusterName == "" {
+				return errors.Errorf("--cluster-name is mandatory with `local` mode")
+			}
```

Comment:
👍 but we also need validation on Universal - meaning there should be a validation of config when CP starts.